### ISSUE TITLE
Why dont change the learning rate?

### ISSUE THREAD

**Issue by wuqiangch** [HarryVolek/PyTorch_Speaker_Verification#14]
In you code ,the learning rate allways keeps 0.01 and does not change during the train process.

**Comment by celikmustafa89** [comment]
In the paper, learning rate is adaptive, changing it "decrease it by half every 30M steps". however, code is same in every iteration. Should I add learning rate schedular? or change the optimizer which has learning_rate_decay parameter? any suggestion, comment, whatever? thank you
